ceph/waiter.py

```diff
@@ -0,0 +1,31 @@
+"""Helper object to encapsulate waiting for timeouts"""
+
+import time
+
+
+class Waiter(object):
```

Comment:
Nice. This is a good.

Could we remain the class i.e. `WaitFor` or `WaitUntil`?

Reply:
Changed class name to `WaitUntil`